### ISSUE TITLE
md-tooltip: interpreted values aren't properly handled for aria-labels

### ISSUE THREAD

**Issue by kbae00** [angular/material#7181]
When the parent of an md-tooltip doesn't have an aria-label, the addAriaLabel function in tooltip.js adds one by setting the aria-label value to "element.text().trim()". In the case of a particular directive I'm working on, the value passed into the directive uses variables. These are fine for the md-tooltip in the directive template, but the aria-label that gets set on the parent element becomes "{{variable1 name}} ({{variable2 name}})" and that is what a screen reader ends up verbalizing: "left brace left brace variable1 name right brace right brace...". I suspect this is happening because of the way the aria-label is being set "element.text().trim()". It would be nice to have the interpreted values work as aria labels and to not have to explicitly set an aria-label on the parent to get around this.

**Comment by ErinCoughlan** [comment]
Duplicate of #6855

**Comment by jelbourn** [comment]
Closed as dupe.
